src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] { new Truck(), new Bulldozer(), new Excavator() };
```

Comment:
you can omit `new Machine[]`

```suggestion
        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator() };
```